loads/loadconf.py

```diff
@@ -1,29 +1,23 @@
 import os
-from pprint import pprint
 
 from loads import (
     DIR_TEMP,
-) 
+)
 from loads.utils import (
    process,
    process_parse
 )
 from outlawg import Outlawg
 
 Log = Outlawg()
+PATH_LOADS_BROKER = '{0}/loads-broker'.format(DIR_TEMP)
 
 
 def load_config(github_owner_repo):
     loads_broker_install()
 
-    # check for loads-broker
-    # - activate venv
-    # - retrieve master githash from github
-    # - git check local githash - if same, then continue
-    # - if not then install!
-    # - pip install -r test-requirements.txt
-    # start loads-broker -i json-file
-    loads_broker_run(github_owner_repo)
+    # Returns plans for the user to select as a list of dicts
```

Comment:
probably should update comment to reflect that we're also passing back the loads-broker proc as well.

# Returns loads-broker process and plans (for the user to select as a list of dicts)